source/blueprints/screening_app.rst

```diff
@@ -0,0 +1,507 @@
+
+
+2. Example Dialogue Interactions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The conversational flows for a health screening will primarily involve asking users a series of questions about their medical history relevant to prediabetes. Once all necessary questions have been asked, the users are informed of their risk for presenting prediabetes.
+
+Here is an example of a scripted dialogue interaction for conversational flows. Although the screening is done in Spanish, for instructional purposes, we demonstrate the same interaction in English.
+
+.. image:: /images/screening_interactions.png
+    :width: 700px
+    :align: center
+
+3. Domain-Intent-Entity Hierarchy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here is the NLP model hierarchy for our screening application.
+
+.. image:: /images/screening_hierarchy.png
+    :height: 600px
+    :align: center
+
+The screening blueprint is organized into two domains: ``greetings`` and ``prediabetes_screening``.
+
+The full list of intents for both domains is illustrated below.
+
+The ``greetings`` domain supports the following intents:
+
+   - ``greet`` — Greet the user and ask them if they would like to be screened for prediabetes.
+   - ``exit`` — Opt-out of the screening and say bye to the user.
+
+The ``prediabetes_screening`` domain supports the following intents:
+
+   - ``opt_in`` — User chooses to undergo the screening.
+   - ``answer_age`` — User wants to provide their age.
+   - ``answer_gender`` — User wants to provide their gender.
+   - ``answer_yes`` — User wants to answer a question with an explicit "yes".
+   - ``answer_no`` — User wants to answer a question with an explicit "no".
+   - ``answer_yes_gestational`` — User wants to answer the question regarding gestational diabetes with an implied "yes". For example, the answer: "During my first pregnancy" implies an affirmative response.
+   - ``answer_yes_family`` — User wants to answer the question regarding family history of diabetes with an implied "yes".
+   - ``answer_yes_hbp`` — User wants to answer the question regarding high blood pressure with an implied "yes".
+   - ``answer_yes_active`` — User wants to answer the question regarding physical activity with an implied "yes".
+   - ``answer_height`` — User wants to provide their height.
+   - ``answer_weight`` — User wants to provide their weight.
+
+There are two types of entities in MindMeld: :ref:`System Entities <system-entities>` and :doc:`Custom Entities <../userguide/entity_recognizer>`. System entities are pre-defined in MindMeld. Examples include ``sys_temperature``, ``sys_time``, and ``sys_interval``. Custom entities are defined by the developers of each application. Within each entity folder, the file ``gazetteer.txt`` contains the full list of values for each custom entity.
```

Comment:
Should we clarify that the gazetteer offers a strong signal and does not mean we can identify only those entity values?